#103: Added UrlPath parameter (#102)
exasol/exasol-driver-go: README.md

#### With Exasol DSN

> You can also create a connection without the builder directly using a string:

Reviewer: ```suggestion
You can also create a connection replacing the builder with a simple string:
```

}
```

> If a value in the connection string contains a `;` you need to escape it with `\;`. This ensures that the driver can parse the connection string as expected.

Reviewer: :+1: 

I wanted to add a paragraph like "In this case, please ensure to escape all values properly."
But your proposal is even better and more concrete. 
Thank you!

### Configuring TLS

> We recommend to always enable TLS encryption. This is on by default, but you can enable it explicitly via driver property `encryption=1` or `config.Encryption(true)`. Please note that starting with version 8, Exasol does not support unencrypted connections any more, so you can't use `encryption=0` or `config.Encryption(false)`.

Reviewer: ```suggestion
We recommend to always enable TLS encryption. This is on by default, but you can enable it explicitly via driver property `encryption=1` or `config.Encryption(true)`. Please note that starting with version 8, Exasol does not support unencrypted connections anymore, so you can't use `encryption=0` or `config.Encryption(false)`.
```

    Use this when the database has a CA-signed certificate. This is the default behavior.
* With `validateservercertificate=1;certificatefingerprint=<fingerprint>` (or `config.ValidateServerCertificate(true).CertificateFingerprint("<fingerprint>")`) you can specify the fingerprint (i.e. the SHA256 checksum) of the server's certificate.

>     This is useful when the database has a self-signed certificate with invalid hostname but you still want to verify connecting to the correct host.

Reviewer: This was a very-very correct host :rofl:
